tests/Performance/webserver/test_scripts/web.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""
+It is used to test the tornado web framework. This can be used:
+-change of an underlying library such as ThreadPool
+-change of the Tornado configuration (running more Tornado processes)
+-Tornado scalability of a certain machine
+"""
+
+import time
+import httplib
+
+
+class Transaction(object):
+
+  def __init__(self):
+    self.custom_timers = {}
+    self.url = "/DIRAC/s:LHCb-Certification/g:lhcb_prmgr/ExampleApp/getSelectionData"
+    self.conn = httplib.HTTPConnection("lhcb-cert-dirac.cern.ch")
```

Comment:
These 2 lines should be somehow configurable.

Reply:
well. This always changes depending what we want to test. How to configure it is another question. We have to see how we setup the performance testing framework and then we can decide how to configure. Maybe use of file or CS or other...